sct-commons/src/test/java/org/lfenergy/compas/sct/commons/scl/ExtRefServiceTest.java

```diff
@@ -418,26 +415,25 @@ void configureNetworkForAllControlBlocks_should_create_GSE_elements() {
         ControlBlockNetworkSettings controlBlockNetworkSettings = controlBlockAdapter -> new Settings(0x1D6, (byte) 4, minTime, maxTime);
 
         // When
-        SclReport sclReport = ExtRefService.configureNetworkForAllControlBlocks(scd, controlBlockNetworkSettings,
-            APP_ID_MIN, APP_ID_MAX, MAC_ADDRESS_MIN, MAC_ADDRESS_MAX, GSE);
+        SclReport sclReport = ExtRefService.configureNetworkForAllControlBlocks(scd, controlBlockNetworkSettings, RANGES_PER_CB_TYPE);
```

Comment:
name of the test method should include both creation of the GSE and SMV elements ?

Reply:
That would be to much assertions in a single test. There is another test that does the same thing (given, when) but checks SMV f`configureNetworkForAllControlBlocks_should_create_SMV_elements`

Reply:
I see, but this action also produces SMV elements.
We can create only one test, we need to add only two assertions and remove the test method for only SMV elements that it also produces GSE elements. ~~configureNetworkForAllControlBlocks_should_create_SMV_elements~~
```suggestion
        SclReport sclReport = ExtRefService.configureNetworkForAllControlBlocks(scd, controlBlockNetworkSettings, RANGES_PER_CB_TYPE);
        ...
        TSMV smv = connectedAP.getSMV().stream()
                .filter(tsmv -> "CB_LD_INST21_SVI".equals(tsmv.getCbName()))
                .findFirst().get();
         assertThat(smv.getLdInst()).isEqualTo("LD_INST21");
          assertThat(smv.getAddress().getP()).extracting(TP::getType, TP::getValue)
              .containsExactlyInAnyOrder(
                      Tuple.tuple("VLAN-PRIORITY", "4"),
                      Tuple.tuple("APPID", "400A"),
                      Tuple.tuple("MAC-Address", "0A-0B-0C-0D-00-FF"),
                      Tuple.tuple("VLAN-ID", "1D6")
              );
```
I don't think there is a need to test each one separately. 
We have about 8 assertions overall that are less than 25.